packages/ra-ui-materialui/src/form/SimpleForm.js

```diff
@@ -70,6 +70,20 @@ const defaultSubscription = {
     invalid: true,
 };
 
+SimpleForm.propTypes = {
+    defaultValue: PropTypes.oneOfType([PropTypes.object, PropTypes.func]), // @deprecated
+    initialValues: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
+    save: PropTypes.func,
```

Comment:
I'm not sure we should export those prop as they are injected by react-admin (save, saving, record and version)